src/_userflair.scss

```diff
@@ -16,108 +16,9 @@
 	overflow: visible !important;
 }
 
-// List service icons
-.flair::before {
-	content: "";
-	width: 0px;
-	height: 0px;
-	background: url("images/general.png") 0 0/128px;
-	border-radius: inherit;
-	margin-left: -16px;
-	transition: inherit;
-}
-
-.flair-MAL::before, 
-.flair-HB::before,
-.flair-ANI::before,
-.flair-AP::before,
-.flair-ann::before,
-.flair-anidb::before,
-.flair-custom::before {
-	width: 16px;
-	height: 16px;
-}
-
-.flair-MAL:hover:before,
-.flair-HB:hover:before,
-.flair-ANI:hover:before,
-.flair-AP:hover:before,
-.flair-ann:hover:before,
-.flair-anidb:hover:before,
-.flair-custom:hover:before {
-	margin-right: 4px;
-}
-
-
 // Expand the flair on hover
 .flair:hover {
 	width: auto;
-	padding-right: 4px;
```

Comment:
After checking the site, I realized there's no space between the emojis and the flair itself, which feels kinda cluttered, and removing this line will also remove the space at the end of the flair, which is more consistent but doesn't really help. With no margin the flairs look like this for me:

![image](https://user-images.githubusercontent.com/4165301/128589524-35d09cf3-095e-4d9a-ba8e-ffc5c7490b5b.png)

And here's what they look like with the 4px spacing on the sides, which I find significantly easier to read and not significantly bulkier:

![image](https://user-images.githubusercontent.com/4165301/128589547-5e207539-6ab5-49f6-812e-de460afeb815.png)


A better solution would be to leave this line in, and add another block that adds back the margin between icon and text that, like so:

```scss
.flair:hover > :last-child {
  margin-left: 5px;
}
```

Since we have tight control over the formatting of flairs with the new system,`> :last-child` to pick out the plain text section of the flair should work; you can also use `> :not([class])` to select *all* text nodes, which shouldn't make a big difference here but may be useful to know in the future.

Reply:
I agree with the above. 

I am also seeing a separate issue on staging where after manually setting from the reddit mod pages, no flair is displayed whatsoever.